### PR TITLE
feat: Healthy node fallback functionality

### LINKED ISSUE #232: Multiple nodes failback
Set up similar node failback method to badger-rewards where if the first node is unavailable, process requests with next node.

Example failback code:
https://github.com/Badger-Finance/badger-rewards/blob/main/config/env_config.py#L90

### PR DESCRIPTION
**Pull request by SHAKOTN**
- Expand chain configs
- Completely replace get_node_url with new function

Fixes #232

### CODE REVIEW

**Comment by btcookies** [comment]
will have to confirm iam roles for each bot on each chain have access to all node secrets, hold merge until confirm

File: scripts/earn.py [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 from src.aws import get_secret
+from src.earner import Earner
+from src.utils import get_healthy_node
 from src.web3_utils import get_strategies_and_vaults
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-INVALID_STRATS = [POLY_OLD_STRATEGY]
+INVALID_STRATS = []
```
Comment (btcookies) [inline scripts/earn.py:13]: why remove this?

Comment (SHAKOTN) [reply scripts/earn.py:13]: Because poly old strategy constant is non-existant now

**Comment by btcookies** [comment]
LGTM waiting on official stamp for iac upgrades

**Pull request merged**